crates/hotshot/types/src/light_client.rs

```diff
@@ -25,7 +25,7 @@ use crate::signature_key::BLSPubKey;
 
 /// Capacity of the stake table, used for light client
 /// TODO(Chengyu): this should be loaded from the sequencer config
```

Comment:
I guess you can remove the TODO comment?

Reply:
yes!
https://github.com/EspressoSystems/espresso-network/pull/3278/commits/e005b62ca2f1668cf32d3d07c4f1ba2e683c7dfb